aidial_adapter_bedrock/utils/adapter_deployments.py

```diff
@@ -0,0 +1,144 @@
+import json
```

Comment:
This module doesn't look like something, that should be in `utils`, I would put this into the root, since it's an important part of application init

Reply:
Fixed. Moved to the root.